### PR TITLE
Update gradle configuration

### PR DESCRIPTION
**Pull request by e5l**
Fix #KTOR-944

### CODE REVIEW

File: app/chat/build.gradle [hunk 1]
```diff
@@ -24,10 +24,10 @@ repositories {
 }
 
 dependencies {
-    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
-    compile "io.ktor:ktor-server-netty:$ktor_version"
-    compile "io.ktor:ktor-websockets:$ktor_version"
-    compile "ch.qos.logback:logback-classic:$logback_version"
-    testCompile "io.ktor:ktor-server-test-host:$ktor_version"
+    api "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
```
Comment (Stexxe) [inline app/chat/build.gradle:27]: Can we use `implementation` instead of `api`?

File: other/reverse-proxy-ws/build.gradle [hunk 2]
```diff
@@ -30,13 +30,13 @@ repositories {
...
 dependencies {
-    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
-    compile "io.ktor:ktor-server-netty:$ktor_version"
-    compile "io.ktor:ktor-client-apache:$ktor_version"
-    compile "io.ktor:ktor-html-builder:$ktor_version"
-    compile "io.ktor:ktor-websockets:$ktor_version"
-    compile "io.ktor:ktor-client-cio:$ktor_version"
-    compile "io.ktor:ktor-client-websockets:$ktor_version"
-    compile "ch.qos.logback:logback-classic:$logback_version"
+    api "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
```
Comment (Stexxe) [inline other/reverse-proxy-ws/build.gradle:33]: I think `implementation` should be enough here

**Review by Stexxe — CHANGES_REQUESTED** [review]

File: app/chat/build.gradle (hunk 1 above)
Comment (e5l) [reply app/chat/build.gradle:27]: it doesn't matter actually: we're not publishing. But let's change it to api->implementation everywhere

**Comment by e5l** [comment]
Fixed, could you take a look?

**Review by Stexxe — APPROVED** [review]

**Pull request merged**